### PR TITLE
test(extension): add tests for LW-9334 and LW-9335

### PR DESCRIPTION
**Pull request by wklos-iohk**
# Checklist

- [X] JIRA - LW-10817
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Explain how does this PR solves the problem stated in JIRA ticket.
You can also enumerate different alternatives considered while approaching this task.

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

Attach screenshots here if implementation involves some UI changes

### CODE REVIEW

File: packages/e2e-tests/src/elements/addNewWallet/MainModal.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef */
+import { ChainablePromiseElement } from 'webdriverio';
+import { OnboardingMainPage } from '../onboarding/mainPage';
+
+class MainModal extends OnboardingMainPage {
```
Comment (ljagiela) [inline packages/e2e-tests/src/elements/addNewWallet/MainModal.ts:5]: Maybe you could rename it to AddNewWalletMainModal or smth like this. When this PO is called later it does not say much what kind of MainModal it is.

File: packages/e2e-tests/src/elements/addNewWallet/MainModal.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  private MODAL = '.ant-modal-content';
+  private CLOSE_BUTTON = '[data-testid="navigation-button-cross"]';
+
+  get modal(): ChainablePromiseElement<WebdriverIO.Element> {
```
Comment (ljagiela) [inline packages/e2e-tests/src/elements/addNewWallet/MainModal.ts:9]: maybe rename to container ? then when called it looks a bit better than MainModal.modal :)

**Review by ljagiela — CHANGES_REQUESTED** [review]

File: packages/e2e-tests/src/elements/addNewWallet/MainModal.ts (hunk 1 above)
Comment (wklos-iohk) [reply packages/e2e-tests/src/elements/addNewWallet/MainModal.ts]: updated

File: packages/e2e-tests/src/elements/addNewWallet/MainModal.ts (hunk 2 above)
Comment (wklos-iohk) [reply packages/e2e-tests/src/elements/addNewWallet/MainModal.ts]: updated

**Review by oldGreg5 — APPROVED** [review]

**Pull request merged**